packages/website/src/features/Router/RouterPage.tsx

```diff
@@ -0,0 +1,220 @@
+version = "0.1"
+description = "Sample Router Project"
+
+[contract.Counter]
+artifact = "Counter"
+
+[contract.AnotherCounter]
+artifact = "AnotherCounter"
+
+[router.Router]
+contracts = [
+  "Counter",
+  "AnotherCounter",
+]
+depends = [
+  "contract.Counter",
+  "contract.AnotherCounter",
+]`;
+
+const code2 = `[setting.admin]
+defaultValue = "0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266"
+
+[provision.proxy]
+source = "transparent-upgradable-proxy:4.9.3"
+options.admin = "<%= settings.admin %>"
+options.implementation = "<%= contracts.Router.address %>"
+options.abi = "<%= contracts.Router.abi %>"
+depends = ["router.Router"]`;
+
+export const RouterPage = () => {
+  const theme = useTheme();
+
+  return (
+    <Flex
+      background={theme.gradients.dark}
+      backgroundAttachment="fixed"
+      flex="1"
+      py={[0, 0, 8]}
+    >
+      <Box
+        p={8}
+        maxW="container.md"
+        m="auto"
+        background="black"
+        border="1px solid"
+        borderColor="gray.700"
+        borderRadius="md"
+      >
+        <Heading
+          as="h2"
+          size="lg"
+          fontWeight={600}
+          pb={4}
+          mb={4}
+          borderBottom="1px solid"
+          borderBottomColor="gray.800"
+          letterSpacing="0.2px"
+        >
+          Use a Synthetix Router
+        </Heading>
+        <Text mb={4}>
+          There is a limit to the size of smart contracts deployed on EVM
+          blockchains. This can create complications during the development of
+          protocols, where engineers may want an arbitrary amount of code to be
+          executable at a single address.
+        </Text>
+        <Text mb={4}>
+          To overcome this constraint, Cannon includes a plug-in for the{' '}
+          <Link
+            isExternal
+            href="https://github.com/synthetixio/synthetix-router"
+          >
+            Synthetix Router
+          </Link>
+          . This can be used by defining a <Code>router</Code> step in
+          Cannonfiles. This accepts an array of contracts and automatically
+          generates a router contract which will delegate calls to them.
+        </Text>
+        <Text mb={4}>
+          In this guide, we’ll walk through{' '}
+          <Link
+            isExternal
+            href="https://github.com/usecannon/cannon/tree/main/examples/router-architecture"
+          >
+            a simple example
+          </Link>{' '}
+          that uses the router and adds a transparent upgradable proxy.
+        </Text>
+
+        <Heading size="md" mb={4} mt={6}>
+          Create a Router
+        </Heading>
+        <Text mb={4}>Start by installing/upgrading Cannon:</Text>
+        <Box mb={4}>
+          <CommandPreview command="npm i -g @usecannon/cli" />
+        </Box>
+        <Text mb={4}>
+          Also make sure you have an{' '}
+          <Link isExternal href="https://docs.ipfs.tech/install/ipfs-desktop/">
+            IPFS node
```

Comment:
technically this is optional now. we have the ability to store objects in local files. it just means that you will be dependent upon public registries for downloading packages, however.